torch/quantization/quantize.py

```diff
@@ -9,14 +9,23 @@
 import torch.nn.quantized as nnq
 import torch.nn.intrinsic.qat as nniqat
 
-from .quantization_mappings import (get_default_dynamic_quant_module_mappings,
-                                    get_default_static_quant_module_mappings,
-                                    get_default_qat_module_mappings,
-                                    get_default_qconfig_propagation_list)
+from .quantization_mappings import (
+    get_default_dynamic_quant_module_mappings,
+    get_default_static_quant_module_mappings,
+    get_default_qat_module_mappings,
+    get_default_qconfig_propagation_list,
+    has_special_act_post_process,
+    get_default_special_act_post_process,
+)
 
 from .stubs import DeQuantStub, QuantWrapper
 from .qconfig import default_dynamic_qconfig, float16_dynamic_qconfig, float_qparams_dynamic_qconfig
 
+def is_activation_post_process(module):
+    return (isinstance(module, torch.quantization.ObserverBase) or
+            isinstance(module, torch.quantization.FakeQuantize) or
+            isinstance(module, torch.quantization.FakeQuantizeBase))
```

Comment:
Do we plan to make FakeQuantize a subclass of FakeQuantizeBase?

Reply:
yes, will do that sometime in the future, maybe before end of this year.